.github/workflows/fetch_versions.yml

```diff
@@ -7,6 +7,7 @@ on:
 permissions:
   contents: write
   pull-requests: write
+  repository-projects: read
```

Comment:
Since the fetchclient.go calls Repositories.ListTags, I thought this might be required?